src/parser/expression.js

```diff
@@ -714,8 +716,17 @@ pp.parseObj = function (isPattern, refShorthandDefaultPos) {
       prop.type = isPattern ? "RestProperty" : "SpreadProperty";
       node.properties.push(prop);
       if (isPattern) {
-        this.expect(tt.braceR);
-        break;
+        const position = this.state.start;
+        if (firstRestLocation !== null) {
+          this.unexpected(firstRestLocation, "Cannot have multiple rest elements in destruction expression");
```

Comment:
I did not use `assignment` as I think this is not an assignment if I'm not mistaken:

```
function ({a,b,...c,...c}) {}
```
But this would issue the same error. Should add a test for it if there isn't one yet.

Can use `"Cannot have multiple rest elements when destructuring"`.